### DISCUSSION TITLE
Does it support android and ios platform? Are there any platform specific changes for android or ios?

### DISCUSSION THREAD

**Discussion by aryamansingh2008** [jeremy-rifkin/cpptrace#64] [Q&A] [answered]
I want to use this library in an android/ios app which uses c++ library. Is it possible to add this library?

**Comment by jeremy-rifkin** [comment, marked as answer]
Hello! Currently I have not tested anything related to android / ios but I'd be happy to support these platforms. There's a small chance the library might "just work" as is, but if not I can try to add support. I'd also happily accept PRs to help with this :)
